examples/default/domain/aggregate.go

```diff
@@ -75,7 +75,7 @@ func (i *InvitationAggregate) HandleCommand(command eh.Command) error {
 			return nil
 		}
 
-		i.StoreEvent(&InviteAccepted{i.AggregateID()})
+		i.StoreEvent(&InviteAccepted{i.AggregateID().(eh.UUID)})
```

Comment:
Here we'll have the same problem of unmarshal `AggregateID` on load Event from db, but there we already use `bson.Raw` and factory to unmarshal the Event.

`eventstore/mongodb/eventstore.go`
```go
func (s *EventStore) Load(id eh.ID) ([]eh.Event, error) {
	....
	var aggregate aggregateRecord
	err := sess.DB(s.db).C("events").FindId(id).One(&aggregate)
	....

	events := make([]eh.Event, len(aggregate.Events))
	for i, record := range aggregate.Events {
		// Create an event of the correct type.
		event, err := eh.CreateEvent(record.EventType) // <--- Factory create an Event with ID instead of an type
		....

		// Manually decode the raw BSON event.
		if err := record.Data.Unmarshal(event); err != nil { // <--- We receive error here
			return nil, ErrCouldNotUnmarshalEvent
		}
		...
	}
}
```
I don't know how to solve this, if you've an idea I really appreciate ;)